xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/htmlheader.vm

```diff
@@ -18,10 +18,11 @@
   #xwikiDataAttribute('wiki', $doc.wiki)##deprecated, use 'reference' instead
   #xwikiDataAttribute('space', $doc.space)##deprecated, use 'reference' instead
   #xwikiDataAttribute('page', $doc.name)##deprecated, use 'reference' instead
-  #xwikiDataAttribute('version', $doc.version)
-  #xwikiDataAttribute('rest-url', $services.rest.url($doc.documentReference))
+  #xwikiDataAttribute('version', $tdoc.version)
+  #xwikiDataAttribute('rest-url', $services.rest.url($tdoc.documentReferenceWithLocale))
   #xwikiDataAttribute('form-token', "$!{services.csrf.token}")
-  #xwikiDataAttribute('user-id', $!services.model.serialize($xcontext.userReference, 'default'))>
+  #xwikiDataAttribute('user-id', $!services.model.serialize($xcontext.userReference, 'default'))
+  #xwikiDataAttribute('locale', $tdoc.realLocale)>
```

Comment:
I'm not an expert here but could you explain the usage of `realLocale` vs `locale`?